canister/src/unstable_blocks.rs

```diff
@@ -240,7 +288,7 @@ mod test {
         let utxos = UtxoSet::new(network);
         let mut forest = UnstableBlocks::new(&utxos, 1, anchor, network);
```

Comment:
Nit: It is always a connected tree, right? So, calling the variable `forest` is no longer appropriate.